src/app/inbox/components/notification-header/notification-header.component.html

```diff
@@ -0,0 +1,39 @@
+<img src="./assets/vectors/orcid.logo.svg" alt="ORCID.org"/>
+<hr class="logo" />
+<p>
+  <ng-container i18n="@@inbox.header.hi">
+    Hi
+  </ng-container>
+    {{ displayName }}
+  <a
+     (click)="navigateTo(baseUri + orcidId)"
+  >
+    ({{baseUri + orcidId}})
+  </a>
+</p>
+<p>
+  <ng-container i18n="@@inbox.header.gotNewNotifications">
+    You've got new notifications on your ORCID record. To see more details for a particular notification or to take action please
+  </ng-container>
+  <a (click)="navigateTo(baseUri + 'inbox')">
```

Comment:
This is a strange header to have. Since is talking about and linking to the same `/inbox` page that the user is currently at. I think the header and footer are only for emails